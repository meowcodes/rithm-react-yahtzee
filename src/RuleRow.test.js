import React from "react";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import RuleRow from "./RuleRow";

// smoke test
it("renders without crashing", function () {
    mount(<RuleRow name="Ones" score={3}/>);
});

// snapshot test
it("matches snapshot", function () {
    let wrapper = mount(<RuleRow name="Ones" score={3}/>);
    let serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
});
