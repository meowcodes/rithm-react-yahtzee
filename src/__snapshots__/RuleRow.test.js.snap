// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<RuleRow
  name="Ones"
  score={3}
>
  <tr
    className="RuleRow RuleRow-active"
  >
    <td
      className="RuleRow-name"
    >
      Ones
    </td>
    <td
      className="RuleRow-score"
    >
      3
    </td>
  </tr>
</RuleRow>
`;
